apps/patient/src/api/internal.ts

```diff
@@ -0,0 +1,162 @@
+import { types } from 'packages/sdk/dist/lib';
+
+import { graphQLClient } from '../configs/graphqlClient';
+import {
+  GET_PHARMACIES,
+  MARK_ORDER_AS_PICKED_UP,
+  REROUTE_ORDER,
+  SELECT_ORDER_PHARMACY,
+  SET_PREFERRED_PHARMACY
+} from '../graphql';
+
+export const AUTH_HEADER_ERRORS = ['EMPTY_AUTHORIZATION_HEADER', 'INVALID_AUTHORIZATION_HEADER'];
+
+/**
+ * Queries
+ */
+
+export const getPharmacies = async (
+  searchParams: {
+    latitude: number;
+    longitude: number;
+    radius: number;
+  },
+  limit: number,
+  offset: number,
+  token: string
+) => {
+  graphQLClient.setHeader('x-photon-auth', token);
```

Comment:
I'm seeing some repetition here, maybe there are some reusable parts? in here... maybe this auth token could be set once and used throughout via a provider? 

Error handling could be standardized, done like this in some places:

```
      const { message } = error.response.errors[0];
      throw new Error(message);
```
or 
``` 
throw new Error(error.response.errors[0].message);
```